ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -342,6 +391,181 @@ objects:
           oc wait namespace/openshift-ssh-bastion --for delete --timeout=10m
         }
 
+        function recover-from-etcd-quorum-loss() {
+          setup_ssh_bastion
+
+          # Machine API won't let the user to destroy the node which runs the controller
+          echo "Finding two masters to destroy"
+          MAPI_POD=$(oc get pod -l k8s-app=controller -n openshift-machine-api --no-headers -o name)
+          SURVIVING_MASTER_NODE=$(oc get ${MAPI_POD} -n openshift-machine-api -o jsonpath='{.spec.nodeName}')
+          mapfile -t MASTER_NODES_TO_REMOVE < <(oc get nodes -l node-role.kubernetes.io/master= -o name | grep -v "${SURVIVING_MASTER_NODE}")
+          MASTER_MACHINES_TO_REMOVE=()
+          for master in ${MASTER_NODES_TO_REMOVE[@]}
+          do
+            MASTER_MACHINES_TO_REMOVE+=($(oc get ${master} -o jsonpath='{.metadata.annotations.machine\.openshift\.io\/machine}' | cut -d '/' -f 2))
+          done
+
+          echo "Destroy two masters"
+          # Scale down etcd quorum guard
+          oc scale --replicas=0 deployment.apps/etcd-quorum-guard -n openshift-machine-config-operator
+
+          for machine in ${MASTER_MACHINES_TO_REMOVE[@]}
+          do
+            retry 10 oc --request-timeout=5s -n openshift-machine-api delete machine ${machine}
+          done
+
+          echo "Confirm meltdown"
+          sleep 30
+          oc --request-timeout=5s get nodes && exit 1
```

Comment:
sleep 30, and get with 5s timeout... That seems too restrictive...

Reply:
Previous discussion [here][1].

[1]: https://github.com/openshift/release/pull/3572#discussion_r286365709

Reply:
`sleep 30` is waiting for machine API to destroy the master, as `oc delete machine` doesn't wait for it be destroyed.

`--request-timeout` is set to 5 seconds as we know etcd won't reach quorum and the API won't answer